frontend/src/screens/channels/IncreaseIncomingCapacity.tsx

```diff
@@ -203,246 +212,265 @@ function NewChannelInternal({
+                                                      Min. channel size:{" "}
+                                                    </span>
+                                                    {new Intl.NumberFormat().format(
+                                                      peer.minimumChannelSize
+                                                    )}{" "}
+                                                    sats
+                                                  </span>
+                                                </div>
+                                              </div>
+                                            </div>
+                                          </Card>
+                                        ))}
+                                    </div>
+                                  </SheetDescription>
+                                </SheetHeader>
+                              </SheetContent>
+                            </Sheet>
+                          </div>
+                        </Card>
+                      </div>
+                    )}
+                  </div>
+                  {order.paymentMethod === "lightning" && (
+                    <NewChannelLightning order={order} setOrder={setOrder} />
+                  )}
+
+                  <div className="mt-2 flex items-top space-x-2">
+                    <Checkbox
+                      id="public-channel"
+                      defaultChecked={order.isPublic}
+                      onCheckedChange={() => setPublic(!order.isPublic)}
+                      className="mr-2"
+                      disabled={
+                        selectedPeer && !selectedPeer.publicChannelsAllowed
+                      }
+                      title={
+                        selectedPeer && !selectedPeer.publicChannelsAllowed
+                          ? "This channel partner does not support public channels."
+                          : undefined
+                      }
+                    />
+                    <div className="grid gap-1 leading-none">
+                      <Label
+                        htmlFor="public-channel"
+                        className="flex items-center gap-2"
+                      >
+                        Public Channel
+                      </Label>
+                      <p className="text-xs text-muted-foreground">
+                        Enable if you need keysend payments, mostly applicable
+                        for podcasters. Otherwise, it’s not recommended.
+                      </p>
+                    </div>
+                  </div>
+                </>
+              )}
+              {channels?.some(
+                (channel) => channel.public !== !!order.isPublic
+              ) && <ChannelPublicPrivateAlert />}
+              {selectedPeer?.note && <ChannelPeerNote peer={selectedPeer} />}
+
+              <DuplicateChannelAlert
```

Comment:
This is shown when it should not be.

Reply:
in which case it was shown? i checked 
1. already had a megalith channel
2. when tried opening second channel with same peer. warning shown
3. when no channels not shown 
4. when selected different channel not shown

this is the expected behaviour no?

Reply:
This used to be rendered only when `showAdvanced` was true, is that still the case?

Reply:
keeping this by default shown in advanced settings